site/content/troubleshooting-guide/index.md

```diff
@@ -67,4 +67,22 @@ AWS.Deploy.Tools, internally uses a variety of different services to host your .
 
 **Why is this happening**: This happens due to a know issue with the AWS Cloud Development Kit (CDK). The CDK is used to AWS.Deploy.Tools under the covers and it cannot cannot access the `$TEMP` directory inside the `$USERPROFILE` path if it contains a whitespace character.
 
-**Resolution**: See [here](../../docs/getting-started/custom-workspace) for guidance on setting a custom workspace that will be used by AWS.Deploy.tools.
\ No newline at end of file
+**Resolution**: See [here](../../docs/getting-started/custom-workspace) for guidance on setting a custom workspace that will be used by AWS.Deploy.tools.
```

Comment:
Please do the following:
1. Give user a little context about CDK and CDK bootstrap and why we use it
2. Restructure your answers. "1) Possible failure reason-> Resolution; 2)  Possible failure reason-> Resolution"
3. Provide links to other helpful documentation, like https://aws.github.io/aws-dotnet-deploy/docs/getting-started/setup-creds/, CDKBoostrap documentation, etc..

Reply:
Done